the-real-instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const CommentWrapper = styled.div`
```

Comment:
Good job using styled-components here.
Try to start putting all styled components in one file and just import the ones you want to use :D